test/indexer/test_hca_indexer.py

```diff
@@ -840,6 +840,24 @@ def test_cell_line_sample(self):
                 self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (child_cell_line)'])
                 self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [config.null_keyword])
 
+    def test_files_content_description(self):
+        self._index_canned_bundle(('ffac201f-4b1c-4455-bd58-19c1a9e863b4', '2019-10-09T170735.528600Z'))
+        hits = self._get_all_hits()
+        # Specify how many files we expect in each index from bundle with 2 sequence files
+        num_files_per_index = {
```

Comment:
This could be made less brittle. We'd have to modify this after adding a new outer entity. Also don't like the name ("per index" is not accurate) and the fact that it's not obvious that the boolean keys represent the `aggregate` flag. Only files and bundles are special, this mapping hides that to some extent.  

```python
if aggregate:
    # bundle aggregates keep individual files
    num_inner_files = 2 if entity_type == 'bundles' else 1
else:
    # one inner file per file contribution
    num_inner_files = 1 if entity_type == 'files' else 2
```

Fewer and shorter lines, more expressive.